src/seedu/addressbook/data/person/_SimilarNameTest.java

```diff
@@ -0,0 +1,32 @@
+package seedu.addressbook.data.person;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class _SimilarNameTest {
+
+	@Test
+	public void SameNameAndSpaces() throws IllegalValueException {
+		Name name = new Name ("Antonia Devina");
+		Name other = name; 
+		assertEquals("The name is the same", true, name.isSimilar(other));
+	}
+	
+	@Test
+	public void SameNameWithDifferentSpaces() throws IllegalValueException {
+		Name name = new Name ("Antonia Devina");
+		Name other = new Name ("AntoniaDevina"); 
```

Comment:
Use @Before to initialize variables above, so you don't need to repeat it in every test case.